src/frontend/src/lib/components/transactions/AllTransactionsList.svelte

```diff
@@ -58,10 +65,8 @@
 
 <!--TODO: include skeleton for loading transactions and remove nullish checks-->
 {#if nonNullish(sortedTransactions) && sortedTransactions.length > 0}
-	{#each transactions as transaction, index (`${transaction.id}-${index}`)}
-		<div in:slide={SLIDE_DURATION}>
-			<svelte:component this={transaction.component} {transaction} token={transaction.token} />
-		</div>
+	{#each Object.entries(groupedTransactions) as [date, transactions], index (`${date}-${index}`)}
```

Comment:
I think with grouping by date, we can be sure that `date` is unique, otherwise it may indicate that "something went wrong".

Reply:
so, the function per se guarantees it, since it returns a `Record`

but if you prefer I can put a pre-check, in the component directly, some sort or guard, not really sure which

Reply:
Yeah, that's exactly what I meant - since `date` is unique, we don't need `index` here. Not a must though, if you still want to keep it

Reply:
ah! understood now! good one! yes, removed!